ui/app/components/masked-input.js

```diff
@@ -0,0 +1,54 @@
+import Ember from 'ember';
+const { computed } = Ember;
+import autosize from 'autosize';
+
+
+export default Ember.Component.extend({
+	value: null,
+	didInsertElement(){
+		this._super(...arguments);
+		autosize(this.element.querySelector('textarea'));
+	},
+	didRender(){
+		this._super(...arguments);
+		autosize.update(this.element.querySelector('textarea'));
+	},
```

Comment:
The `didRender` hook is also fired on initial render. Maybe you want `didUpdate` here? [Component lifecycle hooks docs here](https://guides.emberjs.com/release/components/the-component-lifecycle/)